libs/MailDB/include/MailDB/IMailDB.h

```diff
@@ -59,30 +59,19 @@ class IMailDB
     IMailDB(IMailDB&&) = delete;
     IMailDB& operator=(IMailDB&&) = delete;
 
-    // TODO: Viacheslav
```

Comment:
Remove these comments

Reply:
I deleted the comments, everything is clean in the real file. You just looked at the code line that was marked as removed